### DISCUSSION TITLE
Import from Arc on iOS

### DISCUSSION THREAD

**Discussion by Loskir** [Freika/dawarich#452] [Ideas]
There is a family of location tracking apps on iOS:
- [Arc](https://apps.apple.com/us/app/arc-timeline-trips-places/id1063151918)
- [Arc Mini](https://apps.apple.com/us/app/arc-mini/id1510217238)
- [Arc Timeline Recorder](https://apps.apple.com/us/app/arc-timeline-recorder/id6499296846)

They all function as local-first trackers and store data in iCloud. Would be nice to be able to import its data into Dawarich

**Comment by Freika** [comment]
@Loskir although it would be great to support more client tracking applications, I'm not aware of a way of getting data, stored in iCloud. In Arc Mini's page I see a screenshot featuring export to GPX or GeoJSON, that might be the best option.
